### DISCUSSION TITLE
Charge is constantly under 20%

### DISCUSSION THREAD

**Discussion by centuryblack9** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1219] [General]
I recently bought Air M1 and installed AlDente. I set the limit to 70% but I never saw it at 70. Most of the times it is under 20%. Since yesterday it is at 16%. I work whole day at home and keep charger plugged in. But if I want to go out and work, I can't do it with 16% charge. It charge for some time and stops but never reaching 70%.

Limit: 70%
Enabled features:
Stop charging when sleeping
Stop charging when app closed
Sailing Mode (Difference 10%)
Heat Protection (Max temp: 33)

Please advice. Thank you.

**Comment by MatthiasKerbl** [comment]
Hi @centuryblack9,

I am sorry to hear you are having issues with AlDente.
If possible, please share a debug file the next time this happens so that we can assess this issue better. You can find a guide on [How to generate and share a debug file](https://apphousekitchen.com/how-to-generate-and-share-a-debug-file/) on our blog.

Best,
Matthias

**Reply by centuryblack9** [reply]
Hi Matthias, thanks for the quick response. Somehow I can't attach the .pasta file here. It's greyed out. I've saved it here:

https://we.tl/t-PtZaBOBWLT

Now the charge is stuck at 15% and the power source is adapter and 'Battery is not charging'. Please advice.

Thanks

**Reply by MatthiasKerbl** [reply]
Hi @centuryblack9,

Thank you for the debug file.
It looks like your Heat Protection Limit is too low at 33 °C and your Battery is above that limit most of the time and therefore does not charge properly. Please try to deactivate the Heat Protection feature or set it higher to 40 °C for example.

Best,
Matthias

**Reply by centuryblack9** [reply]
Thanks Matthias, it worked :)
